example/metric/exponential_histogram/example.go

```diff
@@ -0,0 +1,322 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/rand"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"gonum.org/v1/gonum/stat"
+	"gonum.org/v1/gonum/stat/distuv"
+
+	mexporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric"
+
+	"go.opentelemetry.io/contrib/detectors/gcp"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+	sdkmetric "go.opentelemetry.io/otel/sdk/metric"
+	"go.opentelemetry.io/otel/sdk/metric/metricdata"
+	"go.opentelemetry.io/otel/sdk/resource"
+	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
+)
+
+func workloadMetricsPrefixFormatter(d metricdata.Metrics) string {
+	return fmt.Sprintf("custom.googleapis.com/%s", d.Name)
```

Comment:
does the example need to write to custom.googleapis.com?  Did workload.googleapis.com not work?

Reply:
I don't think it "needs" to write to `custom.googleapis.com`. I remember trying to use `workload.googleapis.com` and then not seeing the metrics, but this probably was another bug during development. I never tried `workload.googleapis.com` again.

I will try again with `workload`, is there a preference ? I don't really know the main difference.

Reply:
Used `workload.googleapis.com` and it worked fine 👍 . Fixed.